docs/source/science/ATL08/ATL08.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>Should we say more about this being a Cloud oriented method?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/362/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>"NSIDC DAAC S3 and NASA's Earthdata search" this is confusing, both methods rely on EarthData Search - one method uses cloud access over S3, the other uses traditional https downloading.</p>